linera-storage/src/db_storage.rs

```diff
@@ -661,6 +679,35 @@ where
     C: Clock,
     Store::Error: Send + Sync,
 {
+    fn construct_query(hashes: &[CryptoHash]) -> Result<Vec<Vec<u8>>, ViewError> {
```

Comment:
Pretty minor, but I would actually rename this to `get_keys` or something like that: we're not actually constructing the query here, we're just getting the keys